test_workloads.py

```diff
@@ -157,7 +157,7 @@ def test_nodejs_workload(self):
         assert("Success 1/1" in nodejs_contents)
 
     @pytest.mark.examples
-    @pytest.mark.skipif((os_release_id in ["alpine"] or (node_label == "graphene_22.04_5.19")),
+    @pytest.mark.skipif((os_release_id in ["alpine"] or (node_label == "graphene_22.04_5.19") or (base_os == "ubuntu24.04")),
```

Comment:
Merge the base_os check for alpine and ubuntu 24.04

Reply:
Modified